desktop/app/app-state.js.flow

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+
+export type State = {
+  x: ?number,
+  y: ?number,
+  width: number,
+  height: number,
+  windowHidden: boolean,
+  isMaximized: ?boolean,
+  isFullScreen: ?boolean,
+  displayBounds: ?any,
+  tab: ?string,
+}
+
+export type Options = {
+  path: string,
+  eventHandlingDelay: number,
```

Comment:
Maybe there should be two types, one for the constructor options and the other for this:

then it would look like 

```
export type Options = { 
path...
eventHandlingDelay...
} & ConstructorOptions
```

Because this has an error in that you aren't passing path or eventHandlingDelay when you construct AppState (you don't get an error because app/main-window isn't flow typed